ERCS/erc-7715.md

```diff
@@ -46,6 +46,40 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 
 We introduce a `wallet_grantPermissions` method for the DApp to request the Wallet to grant permissions.
 
+#### Permissions & Policies
+
+Before we specify the JSON-RPC request and response it's important to clarify the relationship between permissions and policies.
+
+Permissions represent a singular piece of authority that can be used by the application.
+eg. ability to transfer a token, update an ENS domain, vote on a DAO, etc
+
+Policies reduce the permitted scope on any given permission. A policy without a permission could be excessively permissive.
+eg. a token allowance, a gas limit, an access list, an ENS record, etc.
+
+For each permission there will be a set of policies and for each policy there will be a set of properties which are specified as follows:
+
+```typescript
+interface TypeDescriptor {
+  name: string;
+  description?: string;
+  [key: string]: string;
+}
+
+interface Permission {
+  type: TypeDescriptor;
+  policies: Policy[];
+  required: boolean;
+  properties: any;
+  data: any;
+}
+
+interface Policy {
+  type: TypeDescriptor;
+  properties: any;
+  data: any;
+}
+```
```

Comment:
I would add here:

```
Some policies may be "global policies," which are not tied to any specific permissions and may apply to any permission.  Examples include gas limits, value limits, target contract, etc.
```

Reply:
Wouldn't it be easier to set those under a Global Permission? If yes, what would you name the permission type? It could be simply "global"